specification/security/resource-manager/common/v1/types.json

```diff
@@ -169,7 +169,8 @@
           "description": "The platform where the assessed resource resides",
           "enum": [
             "Azure",
-            "OnPremise"
+            "OnPremise",
+            "OnPremiseSql"
```

Comment:
>OnPremiseSql [](start = 13, length = 12)

The PR appears to be modifying some definitions shared across security swaggers. Which API versions will this affect?

Reply:
Microsoft.Security/subassessments 2019-01-01-preview and 2020-01-01

Reply:
Your change would effectively alter the behavior of 2 API versions. As per https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md, this is not allowed. You should bump the API version and only allow the new enum values in that API version.

Reply:
I added a new API version "2020-07-01-preview"

Reply:
My change doesn’t affect the API behavior – I didn’t change any of the current fields (not rename any field and not removed any field) I only added a new option for a new value that field can get, this field already exists.